aws/v11.4.1/README.md

```diff
@@ -0,0 +1,80 @@
+# :zap: Giant Swarm Release v11.4.1 for AWS :zap:
+
+This release re-activates the recent AWS [release of high-availability (HA) masters](https://github.com/giantswarm/releases/tree/master/aws/v11.4.0) while fixing issues with certain OIDC configurations.
```

Comment:
```suggestion
This release re-activates the recent AWS [release of high-availability (HA) masters](https://github.com/giantswarm/releases/tree/master/aws/v11.4.0), fixing OIDC configurations issues.
```

Very minor and subjective suggestion to make it clearer that we fixed OIDC thingies _in order to_ reactivate the release.